code/modules/mob/dead/observer/orbit.dm

```diff
@@ -18,94 +34,132 @@
 
 /datum/orbit_menu/ui_act(action, list/params, datum/tgui/ui, datum/ui_state/state)
 	. = ..()
+
 	if(.)
 		return
+
 	switch(action)
 		if ("orbit")
 			var/ref = params["ref"]
-			var/atom/movable/poi = (locate(ref) in GLOB.mob_list) || (locate(ref) in GLOB.poi_list)
-			if (poi == null)
-				. = TRUE
-				return
-			owner.ManualFollow(poi)
-			owner.reset_perspective(null)
+			var/auto_observe = params["autoObs"]
```

Comment:
Please just name it auto_observe, like how every other param matches their variable names.

Reply:
Done.